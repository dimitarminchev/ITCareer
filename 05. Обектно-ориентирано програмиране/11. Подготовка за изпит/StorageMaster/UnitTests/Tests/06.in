RegisterStorage DistributionCenter AmazonDistribution
RegisterStorage Warehouse AmazonWarehouse
AddProduct HardDrive 70
AddProduct HardDrive 60
AddProduct HardDrive 110
AddProduct Gpu 600
SelectVehicle AmazonDistribution 0
LoadVehicle SolidStateDrive
LoadVehicle HardDrive HardDrive HardDrive
SendVehicleTo AmazonDistribution 0 AmazonWarehouse
GetStorageStatus AmazonWarehouse
UnloadVehicle AmazonWarehouse 3
GetStorageStatus AmazonWarehouse
END
