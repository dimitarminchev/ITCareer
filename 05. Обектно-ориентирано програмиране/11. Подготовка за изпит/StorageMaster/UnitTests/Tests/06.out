Registered AmazonDistribution
Registered AmazonWarehouse
Added HardDrive to pool
Added HardDrive to pool
Added HardDrive to pool
Added Gpu to pool
Selected Van
Error: SolidStateDrive is out of stock!
Loaded 2/3 products into Van
Sent Van to AmazonWarehouse (slot 3)
Stock (0/10): []
Garage: [Semi|Semi|Semi|Van|empty|empty|empty|empty|empty|empty]
Unloaded 2/2 products at AmazonWarehouse
Stock (2/10): [HardDrive (2)]
Garage: [Semi|Semi|Semi|Van|empty|empty|empty|empty|empty|empty]
AmazonWarehouse:
Storage worth: $170.00
AmazonDistribution:
Storage worth: $0.00
